wireapp/wire-android-reloaded
# fix(calling): video stays paused after moving the app into foreground (AR-1825)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1825" title="AR-1825" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-1825</a>  1:1 calling - Take app to background and foreground again, video stays paused
  </td></table>
  <br />
 












----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

In an ongoing video call, if the user puts the app in background and then move it back into foreground self video stays paused

### Solutions

We weren't getting the correct app lifecycle

### Dependencies (Optional)

Needs to be merged with
[fix(calling): self user video stream is not sent during a call 
](https://github.com/wireapp/kalium/pull/738)

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

- Start a call
- Turn on your camera
- Put the app in the background and the return it back into foreground
- camera should be turned on

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
